cookbooks/aws-parallelcluster-platform/recipes/config.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+include_recipe 'aws-parallelcluster-platform::openssh'
```

Comment:
Nitpick: maybe we can work on the naming, putting verbs instead of nouns? For instance:
- it's not clear what `openssh` actually does
- `cluster_user` is actually `create_cluster_user`

Reply:
I'm not sure on having the verb before the nouns, especially if there are multiple "actions" for the same recipe. 

I'm probably in favour of having: `cluster_user_setup` or `cluster_user_create`.

Anyway I agree we can find a better naming for these recipes, but we can do it in a separate patch.

Reply:
Usually it's recommended to use verbs.